docs/Learning-Environment-Examples.md

```diff
@@ -246,6 +246,7 @@ If you would like to contribute environments, please see our
     `VisualHallway` scene.
 * Reset Parameters: None.
 * Benchmark Mean Reward: 0.7
+  * To speed up training, you can enable curiosity by adding `use_curiosity: true` in `config/trainer_config.yaml`
```

Comment:
Is the intention to have space prior to the asterisk?

Reply:
yes. It looks all examples have the same list of bullet points (Set-up, Goal,..., Benchmark Mean reward). I want to keep the overall structure, and only add enabling curiosity to achieve the benchmark mean reward faster. So I add this optimization under the benchmark mean reward. Let me know if there is a better place to add a note of enabling curiosity.